docs/README.md

```diff
@@ -9,8 +9,8 @@ to .NET applications without having to modify their source code.
 
 ⚠️ The following documentation refers to the in-development version
 of OpenTelemetry .NET Automatic Instrumentation. Docs for the latest version
-([v0.1.0-beta.1](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/releases/latest))
-can be found [here](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/blob/v0.1.0-beta.1/docs/README.md).
+([0.2.0-beta.1](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/releases/latest))
```

Comment:
Could you please add this to release procedure?

Before release - remove this statement, make a release, crate a tag.
After release, bring back statement with the new link?

Reply:
I think this should be good enough for now: https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/981/commits/b8cc37d15bcc41b8fba5803478755b5030299b8e

We can always refine the docs as a separate PR.